docs/general/builtins/arithmetic.rst

```diff
@@ -17,11 +17,12 @@ Operator   Description
 ``*``      Multiply the first number with the second
 ``/``      Divide the first number by the second
 ``%``      Finds the remainder of division of one number by another
+``^``      Finds the exponentiation of one number raised to another
 ========   =========================================================
 
 Here's an example that uses all of the available arithmetic operators::
 
-    cr> select ((2 * 4.0 - 2 + 1) / 2) % 3 AS n;
+    cr> select ((2 * 4.0 - 2 ^ 3 + 1) / 2) % 3 AS n;
```

Comment:
I think it would be worth adding another example that shows the precedence, which is different from math practice.

Something like:


```diff
diff --git a/docs/general/builtins/arithmetic.rst b/docs/general/builtins/arithmetic.rst
index 3936be7f5ad..8e70310445a 100644
--- a/docs/general/builtins/arithmetic.rst
+++ b/docs/general/builtins/arithmetic.rst
@@ -42,9 +42,24 @@ be an integer with the fractional part truncated::
     +---+-----+
     | 2 | 2.5 |
     +---+-----+
     SELECT 1 row in set (... sec)
 
+
+The ``^`` operator associates left to right by default. This behavior is
+different from typical mathematical practice::
+
+
+    cr> SELECT 2 ^ 3 ^ 3 as a, 2 ^ (3 ^ 3) as b;
+    Got:
+    +-------+-------------+
+    |     a |           b |
+    +-------+-------------+
+    | 512.0 | 134217728.0 |
+    +-------+-------------+
+    SELECT 1 row in set (... sec)
+
+
 .. NOTE::
 
     The same restrictions that apply to :ref:`scalar functions
     <scalar-functions>` also apply to arithmetic operators.
```

Reply:
I will remove this - for the operators of the same precedence it's also from left to right in math, so no difference.
And precedence for mixed types is correct (power comes before */) and we have a test for that
\https://en.wikipedia.org/wiki/Order_of_operations